src-js/lang.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * JS related to our language switcher in the footer/sidebar
+ */
+
+function getElById(id) {
+  return document.getElementById(id)
+}
+
+var languages = {
+  "en": {name: "English", url: "docs/2.0/"},
+  "ja": {name: "日本語", url: "docs/ja/2.0/"}
+}
+
+function reloadWithNewLocale(langCode) {
+  window.location.href = "https://circleci.com/" + languages[langCode].url
+}
+
+// Sets the sidebar language picker to the currently selected language
+function handleSetLanguageOnLoad() {
+  var currentLang = window.currentLang; // current lang is set in _includes/sidebar.html
+  var langSelect = getElById("sidebarLangSelect");
+  for(var i, j = 0; i = langSelect.options[j]; j++) {
```

Comment:
just a note: there may be a slightly more efficient way to do this with the `Array.forEach()` function.